mod_reforged/actor_tooltip_functions.nut

```diff
@@ -45,7 +45,7 @@
 {
 	local currentProperties = _actor.getCurrentProperties();
 	if (!_actor.getCurrentProperties().IsAffectedByReach)
-		return "";
+		return format("<span class='rf_tacticalTooltipReach'>%s - (%s -, %s -)</span>", reachImg, reachAtkImg, reachDefImg);
```

Comment:
Have you considered an `Unaffected` with just the main reach icon in front of it?

Reply:
Yeah I think I proposed something like that to Taro (I proposed the word "Irrelevant" as we had on the Reach status effect for such entities) and he ended up going with this `-` design maybe because the word didn't look as good but he didn't explain.

Reply:
I thought using a "-" to signify irrelevance / not affected is rather common practice, and with this the layouting stays consistent.